src/System.Private.CoreLib/shared/System/Globalization/TextInfo.cs

```diff
@@ -13,19 +13,29 @@
 ////////////////////////////////////////////////////////////////////////////
 
 using System.Diagnostics;
+using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 using System.Runtime.Serialization;
 using System.Text;
+using Internal.Runtime.CompilerServices;
+
+#if BIT64
+using nuint = System.UInt64;
+using nint = System.Int64;
+#else // BIT64
+using nuint = System.UInt32;
+using nint = System.Int32;
+#endif // BIT64
 
 namespace System.Globalization
 {
     public partial class TextInfo : ICloneable, IDeserializationCallback
     {
         private enum Tristate : byte
```

Comment:
Should the underlying type rather be `sbyte` since you are taking advantage of the negative value?

Reply:
Per your other feedback, no longer using the negative value, so this is moot.